src/tests/support.c

```diff
@@ -180,6 +180,18 @@ delete_recursively(const char *path)
     nftw(path, remove_cb, 64, FTW_DEPTH | FTW_PHYS);
 }
 
+void copy_recursively(const char *src, const char *dst) {
+    /* sanity check, we should only be copying things to /tmp/ */
+    assert_int_equal(strncmp(dst, "/tmp/", 5), 0);
+    assert_true(strlen(dst) > 5);
+
+
+    // TODO: maybe use fts or something less hacky
+    char buf[2048];
+    snprintf(buf, sizeof(buf), "/bin/cp -ar '%s' '%s'", src, dst);
```

Comment:
Fails on macOS:
```
cp: the -R and -r options may not be specified together.
Could not run the test - check test fixtures
[  ERROR   ] test_ffi_detect_key_format
```

Reply:
Should be OK now, the `-r` was redundant...